foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_7_0/ProjectSpecificImageNameMigration.java

```diff
@@ -0,0 +1,64 @@
+package com.boozallen.aissemble.upgrade.migration.v1_7_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractAissembleMigration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.technologybrewery.baton.BatonException;
+
+import java.io.File;
+import java.io.IOException;
+
+import static com.boozallen.aissemble.upgrade.util.FileUtils.*;
+
+public class ProjectSpecificImageNameMigration extends AbstractAissembleMigration {
```

Comment:
A: Recommend adding a class level comment describing what the migration aims to accomplish.